package.json

```diff
@@ -18,7 +18,7 @@
     "start": "NODE_ENV=development node server.js",
     "build": "next build",
     "analyse": "NODE_ENV=analyse next build",
-    "prod": "NODE_ENV=production node server.js",
+    "prod": "NODE_ENV=development node server.js",
```

Comment:
why?

Reply:
There is a bug in staging that I tried to fix, I figured out what the problem is and it's done now.